init.go

```diff
@@ -33,3 +33,9 @@ func StartTest(t *testing.T, opts ...scopetesting.Option) *scopetesting.Test {
 	pc, _, _, _ := runtime.Caller(1)
 	return scopetesting.StartTestFromCaller(t, pc, opts...)
 }
+
+// Instruments the given benchmark
```

Comment:
It would be great to add here as a comment what we put in the PR explaining the usage. It will be rendered by `godoc`

Reply:
done